packages/vant/src/image-preview/ImagePreviewItem.tsx

```diff
@@ -43,6 +45,23 @@ const bem = createNamespace('image-preview')[1];
 
 const longImageRatio = 2.6;
 
+const imagePreviewItemProps = {
+  src: String,
+  show: Boolean,
+  active: Number,
+  minZoom: makeRequiredProp(numericProp),
+  maxZoom: makeRequiredProp(numericProp),
+  rootWidth: makeRequiredProp(Number),
+  rootHeight: makeRequiredProp(Number),
+  disableZoom: Boolean,
+  doubleScale: Boolean,
+  closeOnClickOverlay: Boolean,
+};
+
+export type ImagePreviewItemProps = ExtractPropTypes<
+  typeof imagePreviewItemProps
+>;
+
 export default defineComponent({
   props: {
```

Comment:
We can use `imagePreviewItemProps` 😄

```suggestion
  props: imagePreviewItemProps
```